tutorials/transfers/index.md

```diff
@@ -0,0 +1,271 @@
+         - **perBlock** - the amount of funds that will be released per block. In `cliff vesting`, the funds will be released all at once after the specified time period, so this value should be equal to the locked amount. In `linear vesting`, the funds will be released linearly over the specified time period, so this value should be calculated based on the total amount of funds and the duration of the vesting period.
+         - **startingBlock** - the block number when the funds will be released  
+       3. Click on the **Submit Transaction** button to proceed 
+
+    ![Vested Transfer Form](/images/tutorials/transfers/vested-transfers/vested-transfers-3.webp)
+
+    !!! note
+        The above image shows how to perform a cliff vesting transfer, since **locked** and **perBlock** are the same. If you want to perform a linear vesting transfer, you should calculate the **perBlock** value based on the total amount of funds and the duration of the vesting period. For example, if you want to release 100 DOT over a period of 10 blocks, the **perBlock** value should be 10 DOT.
+
+4. After submitting the transaction, a pop-up window will appear to confirm the transfer. Click on the **Sign and Submit** button to authorize the transaction:
+
+    ![Vested Transfer Confirmation](/images/tutorials/transfers/vested-transfers/vested-transfers-4.webp)
+
+5. To confirm that the transaction was successful, you can navigate to the **Accounts** tab and check the **locked** balances of the receiver's account (if the **startingBlock** has not been reached yet) or the **vested** balances (if the **startingBlock** has been reached):
+
+    ![Accounts Tab - Locked Balances](/images/tutorials/transfers/vested-transfers/vested-transfers-5.webp)
+
+#### Lazy Vesting
+
+Vesting is lazy, meaning that someone must explicitly call an extrinsic to update the lock placed on an account.
+
+- The `vest` extrinsic will update the lock that is placed on the caller.
+- The `vest_other` will update the lock on another "target" account's funds.
+
+These extrinsics are exposed from the Vesting pallet.
+
+If you are using the Polkadot.Js UI, when there are DOT available to vest for an account, then you will have the ability to unlock DOT that has already vested from the Accounts page.
+
+#### Calculating When Vesting Tokens Will Be Available
+
+You can check the status of your DOT vesting by reviewing your accounts, which will show when the vesting period ends. However, some DOT vesting involves "cliffs," where all the tokens are released simultaneously in a single block rather than gradually over time. In these cases, you'll need to directly query the chain state to determine when your tokens will be available, as the vesting period hasn’t technically begun yet—it will occur entirely in one future block.
```

Comment:
```suggestion
You can check the status of your DOT vesting by reviewing your accounts, which will show when the vesting period ends. However, some DOT vesting involves "cliffs", where all the tokens are released simultaneously in a single block rather than gradually over time. In these cases, you'll need to directly query the chain state to determine when your tokens will be available, as the vesting period hasn’t technically begun yet—it will occur entirely in one future block.
```